components/views/works/Works.module.scss

```diff
@@ -1,113 +1,82 @@
         justify-content: center;
-        height: $banner-height;
-        padding-top: 2rem;
-    }
 
-    .contactRow {
-        height: 7.65vh;
-        background-color: var(--sienna500);
-        color: var(--off-white);
-        padding: 1rem;
-        display: flex;
-        align-items: center;
-        justify-content: center;
-        font-weight: bold;
-        font-size: 1.3rem;
-        letter-spacing: 0px;
+        h1 {
+            font-size: 2.5rem;
+            text-align: center;
+            font-weight: 500;
+        }
 
-        a {
-            @keyframes grow {
-                from {
-                    text-shadow: 1.2rem;
-                }
-                to {
-                    font-size: 1.4rem;
+        p {
+            font-size: 1.5rem;
+            font-weight: 400;
+
+            a {
+                color: var(--seagreen500);
+
+                &:hover {
+                    color: var(--skyblue400);
                 }
             }
+        }
 
-            animation: grow 0.75s ease-in-out 6 alternate;
-            margin-right: 10px;
+        @media screen and (max-width: 575px) {
+            padding: 2rem;
         }
     }
 
     .projectsDisplay {
-        display: grid;
-        grid-template-columns: repeat(auto-fit, minmax(25rem, 1fr));
-    }
-}
-
-.WorkCard {
-    position: relative;
+        padding: 0;
+        list-style: none;
 
-    img {
-        width: 100%;
-        height: 100%;
-        object-fit: cover;
-    }
-
-    .cardOverlay {
-        cursor: pointer;
-        opacity: 1;
-        display: flex;
-        flex-direction: column;
-        justify-content: center;
-        align-items: center;
-        padding: 2rem;
-        gap: 1rem;
-        text-align: justify;
-        position: absolute;
-        top: 0;
-        bottom: 0;
-        left: 0;
-        right: 0;
-        height: 100%;
-        width: 100%;
-        opacity: 0;
-        transition: 0.5s ease;
-        background-image: linear-gradient(var(--seagreen400), var(--slate400));
-
-        * {
-            color: var(--off-white);
+        li {
+            width: 48vw;
+            padding: 0 1.5rem;
         }
 
-        p {
-            font-size: 1.2rem;
-            text-align: justify;
+        li:nth-of-type(odd) {
+            float: left;
+            padding-right: 0.25rem;
+        }
+        li:nth-of-type(even) {
+            float: right;
+            margin-top: 4rem;
+            padding-left: 0.25rem;
         }
-    }
 
-    &:hover {
-        .cardOverlay {
-            opacity: 0.9;
-            cursor: inherit;
+        .imgDisplay {
+            padding-top: 55%;
+            background-repeat: no-repeat;
+            background-size: cover;
+            background-position: 50% 50%;
+            position: relative;
         }
-    }
 
-    &.activeLink:hover {
-        .cardOverlay {
-            opacity: 0.9;
-            cursor: pointer;
+        @media screen and (max-width: 750px) {
```

Comment:
Can we make the media widths variables ? Or is that not s thing ?

Reply:
yea